contracts/DAOTreasury.cdc

```diff
@@ -126,9 +126,41 @@ pub contract DAOTreasury {
     }
 
     destroy() {
-      destroy self.multiSignManager
-      destroy self.vaults
-      destroy self.collections
+      // Check if Valuts are empty
+      var isValutEmpty = true
```

Comment:
slight misspelling, should be `isVaultEmpty`

Reply:
nice catch! 😄